{"uuid": "c7bfb4ad-58ab-4f23-84db-fbf21a8cfe7b", "stdout": "fatal: [192.168.122.203]: UNREACHABLE! => {\"changed\": false, \"msg\": \"Failed to connect to the host via ssh: Warning: Permanently added '192.168.122.203' (ECDSA) to the list of known hosts.\\r\\ndan@192.168.122.203: Permission denied (publickey,gssapi-keyex,gssapi-with-mic,password).\\r\\n\", \"unreachable\": true}", "counter": 4, "pid": 16069, "event_data": {"play_pattern": "all", "play": "all", "task": "Gathering Facts", "task_args": "_ansible_version=2.5.2, _ansible_selinux_special_fs=['fuse', 'nfs', 'vboxsf', 'ramfs', '9p'], _ansible_no_log=False, gather_timeout=10, _ansible_module_name=setup, _ansible_verbosity=0, _ansible_syslog_facility=LOG_USER, _ansible_socket=None, gather_subset=all, _ansible_diff=False, _ansible_debug=False, _ansible_shell_executable=/bin/sh, _ansible_check_mode=False, _ansible_tmpdir=None", "remote_addr": "192.168.122.203", "res": {"msg": "Failed to connect to the host via ssh: Warning: Permanently added '192.168.122.203' (ECDSA) to the list of known hosts.\r\ndan@192.168.122.203: Permission denied (publickey,gssapi-keyex,gssapi-with-mic,password).\r\n", "unreachable": true, "changed": false}, "pid": 16069, "play_uuid": "54e1ada8-887a-cac8-d583-000000000007", "task_uuid": "54e1ada8-887a-cac8-d583-00000000000f", "playbook_uuid": "181e047c-15a9-4660-93f0-3b9e7674a7e0", "playbook": "main.yml", "task_action": "setup", "host": "192.168.122.203", "task_path": "/home/dpivonka/Documents/ansible_runner_testing/ansible/project/main.yml:3"}, "end_line": 5, "created": "2018-06-19T16:41:55.097358", "start_line": 4, "event": "runner_on_unreachable"}